dashboards/quantizer_generator.py

```diff
@@ -0,0 +1,99 @@
+from abc import abstractmethod
+
+import streamlit as st
+from quantizers.quantizer import MidiQuantizer
+from quantizers.absolute_time_quantizer import AbsoluteTimeQuantizer
+from quantizers.relative_time_quantizer import RelativeTimeQuantizer
+
+
+class QuantizerFactory:
+    """
+    Base class for Quantizer objects factory. Makes adding new Quantizers to the dashboard easier.
+    """
+
+    quantizer_desc = ""
+
+    @abstractmethod
+    def select_parameters() -> dict:
+        pass
+
+    @abstractmethod
+    def create_quantizer(quantization_cfg: dict) -> MidiQuantizer:
+        pass
+
+
+class AbsoluteTimeQuantizerFactory(QuantizerFactory):
+    quantizer_desc = """
+    - `pitch`: uses all 88 pitch values
+    - `velocity`: quantization using bins from artifacts/bin_edges.yaml
+    - timing:
+        - `start`: quantizes start into `n_start_bins` bins, evenly distributed across `sequence_duration`.
+        If the piece is longer that `sequence_duration`, all late notes will land in the last bin.
+        - `duration`: quantization using bins from artifacts/bin_edges.yaml
```

Comment:
Some explanation about how those values were calculate would be useful, just pointing the reader to a file full of numbers is not really providing much insight :)